### PR TITLE
Azure Add step to enable AAP subscription in marketplace

### PR DESCRIPTION
**Pull request by ariordan-redhat**
Add a step to emphasise the requirement for a valid subscription.

### CODE REVIEW

File: stories/topics/proc-azure-locate-aap-marketplace.adoc [hunk 1]
```diff
@@ -9,5 +9,10 @@
 . Click *Private Products* from the menu on the left of the screen.
 . Search for {PlatformName}.
 . Click the card that is returned in the search. Be sure to select the official offering from Red Hat.
-. Click *Get it Now*, *Continue*, and then *Create* to initiate the deployment process.
+. Click *Get it Now*.
+. Click *Continue*.
+. The *Overview* tab contains important information about activating your subscription for {PlatformNameShort}.
++
+Read the description and follow the link to enable your subscription. You cannot use {PlatformNameShort} without a valid subscription.
```
Comment (tznamena) [inline stories/topics/proc-azure-locate-aap-marketplace.adoc:16]: Here I would add that they should read the whole "Before you begin" section and follow the "Click here" link.

File: stories/topics/proc-azure-provisioning-aap.adoc [hunk 2]
```diff
@@ -34,6 +34,11 @@ Keep this resource group isolated from other resource groups, including the _Res
 . Click btn:[Next].
 . Click the *Business continuity* tab.
 . From the *Disaster Recovery* list, select an option to enable or disable disaster recovery.
+. Select the *Deployment* tab.
+. Note the following requirements in the description:
+** To use {PlatformNameShort}, you must have a valid subscription linked to your Red Hat account.
```
Comment (tznamena) [inline stories/topics/proc-azure-provisioning-aap.adoc:39]: To make the docs consistent with the new tab, we might want to add one more bullet here to mention Red Hat account.

**Review by ianf77 — APPROVED** [review]
LGTM

**Review by tznamena — APPROVED** [review]
Looks good.

**Pull request merged**